editor/modes/visual-editor/block-list.js

```diff
@@ -214,6 +231,9 @@ class VisualEditorBlockList extends Component {
 				...blocks.slice( insertionPointIndex + 1 ),
 			]
 			: blocks;
+		const continueWritingClassname = classnames( 'editor-visual-editor__continue-writing', {
+			'show-controls': this.state.showContinueWritingControls,
```

Comment:
Minor: To reduce chance for naming conflicts, might be good to prefix the modifier class with something distinctive for modifiers, like `is-showing-controls`.